config/groovy/common.groovy

```diff
@@ -108,34 +108,20 @@ class common {
         } else if (itemsRetrieved.contains(itemName)) {
             println "We already retrieved $itemName - skipping"
         } else {
+            itemsRetrieved << itemName
```

Comment:
This is one of problems that arise from not working under git since the very beginning - you replaced code from never commits by the old one, as it got accidentally comitted